web/src/buddy_list.js

```diff
@@ -86,6 +88,77 @@ export class BuddyList extends BuddyListConf {
     all_user_ids = [];
     users_matching_view_ids = [];
     other_user_ids = [];
+    tooltips_initialized = false;
```

Comment:
I don't really like this, but I couldn't figure out a different way to initialize the tooltip only once while still taking advantage of shared code through functions like `this.total_subscriber_count`. Would it better to just totally copy that logic and put this in e.g. `click_handlers` with the other tippy code?

Reply:
The usual way to do this would be to have a `buddy_list.initialize` function called by `ui_init.js` -- that's how all our other modules are initialized. You could put that just before or just after `activity_ui.initialize` to control when it happens; seems like before would be more correct?

Oh or another totally reasonably plan would be for `initialize_right_sidebar` in `sidebar_ui.js` to call `initialize_tooltips`.

Reply:
Ah of course, thanks!